catalog/spatial/geocoding/spatial-geocoding-create/src/main/java/org/codice/ddf/spatial/geocoding/create/GeoNamesCreator.java

```diff
@@ -33,6 +33,7 @@ public GeoEntry createGeoEntry(final String line) {
                 .featureCode(fields[7])
                 .population(Long.parseLong(fields[14]))
                 .alternateNames(fields[3])
+                .countryCode(fields[8])
```

Comment:
A lot of magic numbers here. Also, the parseDouble and parseLong can throw a NumberFormatException. Can the input supply bad data that triggers the exception?